frontend/carbondb-details.html

```diff
@@ -49,15 +49,15 @@ <h1 class="ui header float left">
                                 <i class="burn icon"></i><div id="sum_co2eq"></div>
                             </div>
                             <div class="label">
-                                Sum CO2eq
+                                Sum CO2eq <span style="text-transform: none;">(g)</span>
```

Comment:
Does it make sense to create a css class here? You use it in quite a few places.

Reply:
actually there was a CSS class :) changed